src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -75,7 +75,7 @@
      * @param other The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<? super O> other);
```

Comment:
The current mixin is not mixing in the interface directly; it uses `@Interface` provided by mixins.